identity-apps-core/apps/authentication-portal/src/main/webapp/includes/header.jsp

```diff
@@ -109,3 +165,7 @@
         }
     }
 </style>
+
+<script>
+    document.documentElement.setAttribute("dir", "<%= direction %>");
```

Comment:
Shall we add a null check here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7220/commits/2b4f5e240882d86aa2fb38d72b2c90e7941a1a2f